src/components/auth/signupModal.js

```diff
@@ -0,0 +1,68 @@
+import Swal from 'sweetalert2';
+
+const signUpModal = () => {
+  Swal.fire({
+    title: 'Sign Up',
+    html: ` <input id="swal-input1" class="swal2-input" placeholder="First Name" required>
+            <input id="swal-input2" class="swal2-input" placeholder="Last Name" required>
+            <input id="swal-input3" class="swal2-input" placeholder="Email" required>
+            <input id="swal-input4" class="swal2-input" placeholder="Password" required>`,
```

Comment:
Please it's important to leave one field for the signup form because the required fields for the API are: 
- [ ] **Fullname**
- [ ] **Email**
- [ ] **Password**
- [ ] **Admin (Boolean)**

Ps:  It's not necessary to put the `admin field` on the popup when the user is registering, we can still tell in the component if the user is an `admin` or not.

Reply:
Hi @Have-Samuel ,

Thank you for your awesome review. I initially did not have the documentation of the API and its requirements. But, since you have clarified that, I'll go ahead and make the suggested changes.

> HappyCoding